Unix/nits/base/Run.cpp

```diff
@@ -1043,7 +1037,7 @@ bool Run::ParseArgument(_In_ wchar_t *arg)
         }
 
         char buf[MAX_PATH];
-        wcstombs(buf, second, MAX_PATH);
+		strcpy(buf, second);
```

Comment:
The single-char equivalent of wcstombs() is strlcpy(), not strcpy().  But in this case, there is no reason to make another copy.  You should be able to just use "second", and not bother creating "buf"?